banner/styles.py

```diff
@@ -0,0 +1,45 @@
+import inspect
+
+from django.apps import apps
+from django.template.loader import render_to_string
+from django.utils.module_loading import import_module
+
+
+class BaseStyle(object):
+    image_path = "/admin/banner/images/unknown.png"
+    template_name = "banner/inclusion_tags/banner_detail.html"
+
+    def __init__(self, banner):
+        self.banner = banner
+
+    def get_context_data(self, context):
+        context["object"] = self.banner
+        return context
+
+    def render(self, context):
+        context.push()
+        new_context = self.get_context_data(context)
+        result = render_to_string(self.template_name, new_context.flatten())
+        context.pop()
+        return result
+
+
+BANNER_STYLE_CLASSES = []
```

Comment:
Rather look at https://github.com/praekelt/django-composer-prk/blob/develop/composer/admin.py#L31-L38 for showing the styles. A curated list is less effort to deal with than this implementation.

Reply:
Let's go with the customisation. Easy change later if needed :slightly_smiling_face: